packages/DialogActions/README.md

```diff
@@ -0,0 +1,36 @@
+## DialogActions
+
+The `<DialogActions>` component renders three buttons, Save (hasConfirm), Don't Save (hasDecline) and Cancel to provide actions for modals.
+
+### Installation
+
+`> npm install --save @paprika/l10n`
+and
+`> npm install --save @paprika/dialog-actions`
+or
+`> yarn add @paprika/dialog-actions`
+
+### Usage
+
+```js
+import DialogActions from "@paprika/dialog-actions";
+
+<DialogActions />;
+```
+
+### Props
+
+- `className`
```

Comment:
we should remove className, this should be spread via `{...moreProps}` cc @oscarkwan